corehq/motech/repeaters/expression/repeaters.py

```diff
@@ -209,6 +218,52 @@ def available_for_domain(cls, domain):
             and ARCGIS_INTEGRATION.enabled(domain)
         )
 
+    def send_request(self, repeat_record, payload):
+        response = super().send_request(repeat_record, payload)
+        if not is_success_response(response):
```

Comment:
Is this conditional needed? I'm not sure what a non success response from ArcGIS would look like, but if we receive a non success response then I'm assuming we'd either still have "error" in the response which we can format using `_error_response`, or it would be absent and we'll simply return `response` at the end of this function.

Reply:
This is really just to check that calling `response.json()` is going to succeed. (Maybe ArcGIS is unreachable, or we get a response from a proxy, or the credentials in the repeater's ConnectionSettings are wrong or outdated, etc.) We could check in a different way (e.g. `if hasattr(response, 'json')`), or we could just try calling it and catch the exception if we fail. What do you think?

Reply:
Would it make sense to combine this conditional with the one below so it then becomes `if is_success_response(response) and 'error' in response.json()`? This might also make it a bit more clear for debugging to understand what code path was taken depending on the return value given. 

I understand this is also maybe a bit of a nitpick, so I'll leave it to your judgement and I'm happy to leave it as is if you think it makes sense enough as is.